### PR TITLE
Update Fyne and Go along with version number for image.

### PR DESCRIPTION
**Pull request by Bluebugs**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .PHONY: all base android darwin darwin-sdk-extractor freebsd linux windows web
 
 # VERSION is the Fyne Cross Images version
-VERSION := "1.0.0"
+VERSION := "1.1.0"
```
Comment (Bluebugs) [inline Makefile:4]: Not to sure what our version policy for this image is going to be. I went with increment minor whenever a minor update on the dependencies (here go). I would have incremented the patch version if that was the only change. What do you guys think?

Comment (Jacalz) [reply Makefile:4]: I think a patch version seems sensible to me. However, I also don't quite know what exactly would be require a major version bump.

**Comment by Bluebugs** [comment]
Oh, shoot, the checksum didn't get updated.

File: Makefile (hunk 1 above)
Comment (lucor) [reply Makefile:4]: Thinking out loud here... what about something like: 
- patch when we update patch versions for the main tools: i.e. Go, Zig, Fyne CLI
- minor when we update a minor for the main tools or add support to new OSes.

Following the above this release should be a minor.

Alternatively we'd decide to not follow semantic versioning at all and use something like Ubuntu: YY.MM.x

Comment (Bluebugs) [reply Makefile:4]: > Thinking out loud here... what about something like: 
> - patch when we update patch versions for the main tools: i.e. Go, Zig, Fyne CLI
> - minor when we update a minor for the main tools or add support to new OSes.
> 
> Following the above this release should be a minor.

That what make sense to me too. 

> Alternatively we'd decide to not follow semantic versioning at all and use something like Ubuntu: YY.MM.x 

Hum, that's an interesting idea. I like it as in this case the release of container is more a timely activity than a feature release. What do other think?

Comment (Jacalz) [reply Makefile:4]: I like the idea for patch and minor releases proposed above.

I’m not a fan of year based release versions. If I look at a specific release, I find it hard to know which releases came before and which are after a given release. You always have to consult the release page. I would prefer to use semantic versions if it makes sense in this context

Comment (Bluebugs) [reply Makefile:4]: I kept the current numbering as it seems there is quite uncertainty on using YYMM.x format.

**Review by lucor — APPROVED** [review]

**Review by Jacalz — APPROVED** [review]

**Pull request merged**